packages/manager/modules/pci/src/projects/project/sidebar/sidebar.constant.js

```diff
@@ -62,17 +64,19 @@ export const MENU = [
     ],
     translation: 'cloud_sidebar_network',
   },
-  {
-    subitems: [
-      {
-        options: {
-          state: 'pci.projects.project.kubernetes',
-        },
-        translation: 'cloud_sidebar_orchestration_kubernetes',
-      },
-    ],
-    translation: 'cloud_sidebar_orchestration',
-  },
+  Environment.getRegion() !== 'US'
```

Comment:
An other approach would be to add for instance an exclude entry which can contains a list of regions.

Is that make sense to you?

```js
{
  …,
  exclude: ['US'] // or Environment.getRegion(),
  …,
}
```